### PR TITLE
Fix metrics dashboard after migration

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/15473

### CODE REVIEW

File: dashboards/metrics.jsonnet [hunk 1]
```diff
@@ -12,29 +12,29 @@ stdlib.dashboard(
...
     'Time Series',
   ),
   gridPos={x: 0, y: 0, w: 8, h: 9},
 )
 .addPanel(
   stdlib.multiSeriesChart(
-    'Number of Time Series In Prometheus (Per Control Plane)',
-    'sum(avg(prometheus_tsdb_head_series{customer=~"$customer", cluster_id=~"$control_plane"}) by (customer, cluster_id)) by (cluster_id)',
+    'Number of Time Series In Prometheus (Per Installation)',
+    'sum(avg(prometheus_tsdb_head_series{customer=~".*", installation=~".*"}) by (customer, installation)) by (installation)',
```
Comment (JosephSalisbury) [inline dashboards/metrics.jsonnet:29]: Doesn't this break templating?

Comment (QuentinBisson) [reply dashboards/metrics.jsonnet]: Actually, yes. I fixed the templating in another commit

File: dashboards/metrics.jsonnet [hunk 2]
```diff
@@ -12,29 +12,29 @@ stdlib.dashboard(
...
+    'sum(avg(prometheus_tsdb_head_series{customer=~"$customer", installation=~"$management_cluster"}) by (customer, installation)) by (installation)',
     '{{cluster_id}}',
   ),
   gridPos={x: 8, y: 0, w: 8, h: 9}
 )
 .addPanel(
   stdlib.multiSeriesChart(
     'Number of Time Series In Prometheus (Per Customer)',
-    'sum(avg(prometheus_tsdb_head_series{customer=~"$customer", cluster_id=~"$control_plane"}) by (customer, cluster_id)) by (customer)',
+    'sum(avg(prometheus_tsdb_head_series{customer=~"$customer", installation=~"$management_cluster"}) by (customer, cluster_id)) by (customer)',
```
Comment (JosephSalisbury) [inline dashboards/metrics.jsonnet:37]:
```suggestion
    'sum(avg(prometheus_tsdb_head_series{customer=~"$customer", installation=~"$management_cluster"}) by (customer, installation)) by (customer)',
```

File: dashboards/metrics.jsonnet [hunk 3]
```diff
@@ -70,24 +70,24 @@ stdlib.dashboard(
 
 .addPanel(
   stdlib.stackedPercentageChart(
-    'Percentage Time Series (Per Control Plane)',
-    'sum(prometheus_tsdb_head_series{customer=~"$customer", cluster_id=~"$control_plane"}) by (cluster_id) / scalar(sum(prometheus_tsdb_head_series{cluster_id!=""}))',
+    'Percentage Time Series (Per Management Cluster)',
+    'sum(prometheus_tsdb_head_series{customer=~"$customer", installation=~"$management_cluster"}) by (installation) / scalar(sum(prometheus_tsdb_head_series{installation!=""}))',
     '{{cluster_id}}',
```
Comment (JosephSalisbury) [inline dashboards/metrics.jsonnet:75]:
```suggestion
    '{{installation}}',
```

File: dashboards/metrics.jsonnet [hunk 4]
```diff
@@ -70,24 +70,24 @@ stdlib.dashboard(
...
   ),
   gridPos={x: 0, y: 18, w: 8, h: 9}
 )
 .addPanel(
   stdlib.stackedPercentageChart(
     'Percentage Time Series (Per Customer)',
-    'sum(prometheus_tsdb_head_series{customer=~"$customer", cluster_id=~"$control_plane"}) by (customer) / scalar(sum(prometheus_tsdb_head_series{customer!=""}))',
+    'sum(prometheus_tsdb_head_series{customer=~"$customer", installation=~"$management_cluster"}) by (customer) / scalar(sum(prometheus_tsdb_head_series{customer!=""}))',
     '{{cluster_id}}',
```
Comment (JosephSalisbury) [inline dashboards/metrics.jsonnet:83]:
```suggestion
    '{{installation}}',
```

File: dashboards/metrics.jsonnet [hunk 5]
```diff
@@ -70,24 +70,24 @@ stdlib.dashboard(
...
   ),
   gridPos={x: 8, y: 18, w: 8, h: 9}
 )
 .addPanel(
   stdlib.stackedPercentageChart(
-    'Percentage Memory (Per Control Plane)',
-    'sum(aggregation:prometheus:memory_usage{customer=~"$customer", cluster_id=~"$control_plane"}) by (cluster_id) / scalar(sum(aggregation:prometheus:memory_usage{cluster_id!=""}))',
+    'Percentage Memory (Per Management Cluster)',
+    'sum(aggregation:prometheus:memory_usage{customer=~"$customer", installation=~"$management_cluster"}) by (installation) / scalar(sum(aggregation:prometheus:memory_usage{installation!=""}))',
     '{{cluster_id}}',
```
Comment (JosephSalisbury) [inline dashboards/metrics.jsonnet:91]:
```suggestion
    '{{installation}}',
```

**Comment by QuentinBisson** [comment]
Dashboard is uploaded to grafana for testing

**Comment by QuentinBisson** [comment]
@JosephSalisbury do you mind checking the dashboard in cortex to merge this PR? The dashboard is uploaded there

**Comment by JosephSalisbury** [comment]
How do you feel about the three memory related charts? i'm not really sure what they mean now. does it actually make sense to have those per cluster, so we can see high memory usage servers?

**Review by TheoBrigitte — APPROVED** [review]
LGTM

**Comment by JosephSalisbury** [comment]
See https://github.com/giantswarm/giantswarm/issues/15620 for fixing up recording rules, then we can turn the 'Memory Usage Of Prometheus (Per Management Cluster)' and 'Percentage Of Node Memory (Per Management Cluster)' charts to be per-cluster (regardless of MC and WC)

**Comment by QuentinBisson** [comment]
image
Looks legit

**Review by JosephSalisbury — APPROVED** [review]

**Pull request merged**